.github/workflows/test-build-deploy.yml

```diff
@@ -110,7 +110,6 @@ jobs:
   build:
     runs-on: ubuntu-latest
     needs: [gem-test, test]
-    if: github.ref == 'refs/heads/main'
```

Comment:
are you sure this should be removed... as that would mean it would deploy on every push/push with an open PR...  (I can never remember which)